src/components/ErrorView/index.jsx

```diff
@@ -2,7 +2,7 @@ import PropTypes from 'prop-types'
 import { ButtonV2 } from '../Button/V2'
 import { ButtonRow, Dialog, ErrorBox } from '../Layout'
 import { SmartLink } from '../SmartLink'
-import { GLIF_DISCORD, GLIF_TWITTER } from '../../constants'
+import { useEnvironment } from '../../services'
```

Comment:
Should we path directly into this specific service so we don't "pull all of them in"?

Reply:
Will update some of these in a separate PR